bokehjs/src/coffee/mapper/linear_mapper.coffee

```diff
@@ -15,10 +17,7 @@ class LinearMapper extends Model
 
   v_map_to_target: (xs) ->
     [scale, offset] = @get('mapper_state')
-    result = new Float64Array(xs.length)
-    for x, idx in xs
-      result[idx] = scale * x + offset
-    return result
```

Comment:
I'm very concerned about this. Not generating typed arrays whenever we can could have performance implications for *everything*. I would need to see some benchmarks before endorsing this as-is.